samples/nrf9160/nrf_cloud_mqtt_multi_service/README.rst

```diff
@@ -139,22 +142,28 @@ Using the built-in temperature sensor of the `Nordic Thingy:91`_ requires a `dev
 The devicetree overlay file is automatically applied during compilation whenever the ``thingy91_nrf9160_ns`` target is selected.
 The required Kconfig options are implicitly enabled by :ref:`CONFIG_TEMP_DATA_USE_SENSOR <CONFIG_TEMP_DATA_USE_SENSOR>`.
```

Comment:
I think docs folks usually recommend, instead of saying:
```
enabled by :ref:`CONFIG_FOO`
```

saying:
```
enabled by the :ref:`CONFIG_FOO` option
```